packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -358,7 +358,7 @@ export class DdSdkReactNativeConfiguration {
         readonly trackResources: boolean = false,
         readonly trackErrors: boolean = false,
         readonly trackingConsent: TrackingConsent = DEFAULTS.trackingConsent,
-        readonly useAccessibilityLabel: boolean = DEFAULTS.useAccessibilityLabel
+        readonly useAccessibilityLabel: boolean = DEFAULTS.useAccessibilityLabel // eslint-disable-next-line no-empty-function
```

Comment:
I haven't touched any of these constructors yet, but I think this could do with a refactor soon.